courses/fundamentals_of_ada/quiz/limited_extended_return/quiz.rst

```diff
@@ -0,0 +1,18 @@
+.. code:: Ada
```

Comment:
This is the basically the same as the constructor quiz just adding and "extended return" option. Change this to show bad extended return constructs

Reply:
That's true!